src/coreclr/vm/ceemain.cpp

```diff
@@ -962,6 +962,8 @@ void EEStartupHelper()
         }
 #endif // _DEBUG
 
+        // Initialize managed binder. Ensure g_fEEStarted and COM initialized before executing managed code.
+        SystemDomain::System()->PostStartInit();
```

Comment:
Is it correct time to initialize managed binder here? COM components assert for g_fEEStarted if the initialization is invoked earlier.